configparser.go

```diff
@@ -70,7 +70,7 @@ func getNoOptionError(section, option string) error {
 func New() *ConfigParser {
 	return &ConfigParser{
 		config:   make(Config),
-		defaults: newSection("defaults"),
+		defaults: newSection("default"),
 	}
```

Comment:
I suspect this should be `DEFAULT` see https://docs.python.org/3/library/configparser.html#quick-start "__The only bit of magic involves the DEFAULT section which provides default values for all other sections__".

What do you think?  I suspect this is a long-standing bug.

Reply:
I agree, and it should probably be pointed to the constant defaultSectionName for consistency.
https://github.com/bigkevmcd/go-configparser/blob/2d9389b0b70ee10a05c7f447aec307bced8200a0/configparser.go#L14

That said, I'd like to make the properties of the ConfigParser object public so a custom configParser object can be created with a different default section name.  In the case of the AWS SDK, sections are case-sensitive so it would require lower.

I'll draft up these changes.